test-requirements.txt

```diff
@@ -10,3 +10,4 @@ netflix-spectator-py
 kubernetes
 grpcio
 grpcio-tools
+scs==2.0.1
```

Comment:
What's this for?

Reply:
For test, I found that pinning this version of scs was the only way I could get a working venv + pandas.
My best guess is that something broke with latest scs.
Who knows though, could be python, my laptop, macos. I'll undo.